### PR TITLE
Update install docs

### PR DESCRIPTION
**Pull request by miqmago**
Hi, thanks for this great component! After an hour reading different issues, trying different things, I realised my problem, that seems the problem of many other people too due to the related issues:

- I was looking for an awesome scheduling card and I found the https://github.com/nielsfaber/scheduler-card
- I followed the instructions and successfully installed the card
- Was very excited to test it so I went to the HA panel, added a card and omg it worked! But then I saw the message: *scheduler/add Service not found*
- I've started reading different issues, at least 3 or 4. I've found that there was a different repo to open issues, but didn't understand why, I've thought maybe it's a dependency...
- More reading and I've discovered that I should have the integration ready, but was unable to find integration
- Restarted HA, cleaned cache, tried in mobile phone... and nothing

At that point I've discovered that I should install the `scheduler-component` and then Boom!! It worked! Happy, thanks! :)

Basically I think that a clarification on the documentation that the dependency should be installed and integrated in order to work properly could avoid all this issues.

### CODE REVIEW

**Comment by nielsfaber** [comment]
I’m sorry for the trouble.
I am working on improving this.
Thanks for helping out with the documentation!
